gen2-camera-demo/main.py

```diff
@@ -14,11 +15,17 @@
 But like on Gen1, either depth or disparity has valid data. TODO enable both.
 '''
 
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-pcl", "--pointcloud", help="display a square of a given number", default=False, action="store_true")
```

Comment:
this help message should be a bit more meaningful (copypaste?)

Reply:
Oh. I actually copy pasted and noticed while testing and changed it. forgot to push I think will push along with fix for above error.